src/StaticFAQ/breadcrumbs/__test__/CustomBreadcrumbs.test.js

```diff
@@ -14,42 +14,16 @@ describe('CustomBreadcrumbs', () => {
       { title: 'Baggage', id: '2' },
       { title: 'Article' },
     ];
-    global.innerWidth = 601;
-    const wrapper = mount(
-      <WithRouter>
-        <CustomBreadcrumbs breadcrumbs={breadcrumbs} />
-      </WithRouter>,
-    );
-    const breadcbumbsElements = wrapper.find('.breadcrumb');
-    // 6 because there are 2 for mobile 2 for desktop
-    expect(breadcbumbsElements).toHaveLength(2);
-    expect(breadcbumbsElements.first().text()).toBe('Home');
-    expect(breadcbumbsElements.last().text()).toBe('Article');
-  });
-  // make it work with react-responsive
-  xit('should render shorter breadcrumbs', () => {
```

Comment:
this is old, we don't use `...` in breadcrumbs any more, now is the same for desktop and mobile.

Reply:
we show `Article` instead of the article name.